lib/utils/utils.go

```diff
@@ -664,6 +665,39 @@ func ByteCount(b int64) string {
 		float64(b)/float64(div), "kMGTPE"[exp])
 }
 
```

Comment:
Let's move away from lib/utils, utils is a terrible package name.

I suggest `lib/utils/pagination.Iterate` for the package+function name.